### DISCUSSION TITLE
好奇太记以后会开发插件系统吗

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TidGi-Desktop#580] [Ideas]
就是搞一个用来配置太记的插件系统，与tiddlywiki不太相关。主要是配置太记的功能。比如我觉得太记AI就可以独立出来，需要这个功能的用户，就一方面安装太记的插件，另一方面在tiddlywiki里安装相应的插件。

这样太记就不会因为未来增加新功能而变得繁杂。

但又感觉没这个必要。其实主要的担心还是来自于tiddlywiki，比如如果用tiddlywiki开发游戏，我觉得tiddlywiki可能很难支持得了，而太记就可以承接这一功能，把一些相对底层的框架配置打包成插件，再把一些接口的东西暴露成API做成tiddlywiki的插件。

之前好像也有看到过一个webgal的项目，我用原项目的编辑器编辑都感觉很不方便。感觉用tiddlywiki编辑会更不方便。主要是加载不动。英文论坛里好像有人出了一个新的，估计也是打包那个项目的。但我还没实际测试效果如何。

但技术方面的内容不太懂，就有些好奇。

**Comment by dongrentianyu** [comment]
仔细看了一下设置，发现Version v0.10.3的太记已经去掉了AI的配置。我记得以前是需要在c盘里放那个本地大模型运行文件来着。

**Reply by linonetwo** [reply]
对，测试表明本地AI没太大用，比较笨，而且现在AI免费额度太多了，没必要本地了。

**Comment by linonetwo** [comment]
游戏化我也打算外置了，放 wiki 里太大了。还是利用好 Window 本身这个操作系统，运行多个应用为佳。

**Comment by dongrentianyu** [comment]
确实，要是有一个超级tiddlywiki就好了，可以结合底层的那种，加载起来更快，能承载更多东西。
